go-controller/pkg/ovn/controller/apbroute/network_client.go

```diff
@@ -528,72 +537,75 @@ func (nb *northBoundClient) deleteLogicalRouterStaticRoute(podIP, mask, gw, gr s
 // DelHybridRoutePolicyForPod handles deleting a logical route policy that
 // forces pod egress traffic to be rerouted to a gateway router for local gateway mode.
 func (nb *northBoundClient) delHybridRoutePolicyForPod(podIP net.IP, node string) error {
-	if config.Gateway.Mode == config.GatewayModeLocal {
```

Comment:
what's changing in this function?
just indentation fix refactor?

Reply:
yeah just exiting early if the gw mode is not local, rather than putting the entire function logic into an if statement